### DISCUSSION TITLE
On my machine, all apps using WinUI 2 has rendering issue, need help (I'm not sure if this is correct place to ask)

### DISCUSSION THREAD

**Discussion by cdytoby** [microsoft/WindowsAppSDK#1945] [Q&A] [answered]
I'm not sure if this is correct place to ask, I asked one on reddit but no response.

Windows 10, version 21H1

I don't know when or why it happens, it happens sometime after I updated to 21H1 since it says I'm available to upgrade, also after sometime Microsoft Store is updated to newest version with new UI. Then I noticed something is totally wrong with the UI in Microsoft Store, here are the screenshots:

https://imgur.com/a/YHoRzjH

Not just Microsoft Store, I tried multiple things, like Feedback Center, Powertoys, all have same issue. Then through powertoys I realized they're using WinUI 2 framework, so I downloaded the XAML Controls Gallery, as expected, all of them have the same issue. Screenshots already in the above link.

Toggle and others are not so annoying, but the buttons are very very annoying, because unless I press the button, I can't read the text.

What exactly happened and what should I do to fix it?

Also, I sent a feedback in feedback center, but there are no response at all. 
https://aka.ms/AAfdy5b

**Comment by ghost** [comment]
WinUI 1 , WinUI 2 and WinUI 2 Xaml Islands  are NOT part of WindowsAppSDK. 

WinUI 2 suggestions or issues should be reported in WInUI repo https://github.com/microsoft/microsoft-ui-xaml

FeedbackHub is "The Void" and your problem won't even get eyes by posting there, let alone receiving responses back.

**Comment by btueffers** [comment, marked as answer]
Hi @cdytoby, since this topic is not directly related to Windows App SDK a more effective place to file this issue would be the [WinUI repo](https://github.com/microsoft/microsoft-ui-xaml).

**Reply by cdytoby** [reply]
https://github.com/microsoft/microsoft-ui-xaml/issues/6344 

This is exact the issue and I posted one solution there. This topic here is solved.
